Dockerfile

```diff
@@ -24,7 +25,9 @@ RUN curl -o /usr/local/bin/sops -L https://github.com/mozilla/sops/releases/down
 
 RUN curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/${SKAFFOLD_VERSION}/skaffold-linux-amd64 && chmod +x skaffold && mv skaffold /usr/local/bin
 
-# yq
+# yq and jq
+
+RUN wget https://github.com/stedolan/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 -O /usr/bin/jq && chmod +x /usr/bin/jq
```

Comment:
Maybe let's just add one layer for this. (although we could probably use one for all bins installed)